gcloud/bigquery/table.py

```diff
@@ -312,6 +312,30 @@ def _require_client(self, client):
             client = self._dataset._client
         return client
 
+    def _parse_schema_resource(self, info):
+        """Parse a resource fragment into a schema field.
+
+        :type info: mapping
+        :param info: should contain a "fields" key to be parsed
+
+        :rtype: list of :class:`SchemaField`, or ``NoneType``
+        :returns: a list of parsed fields, or ``None`` if no "fields" key is
+                  present in ``info``.
+        """
+        if 'fields' not in info:
+            return None
```

Comment:
What if `info` has other things in it that the user forgot? (I haven't looked at all code paths which invoke this method yet.)